var/spack/repos/builtin/packages/libdrm/package.py

```diff
@@ -34,6 +34,9 @@ class Libdrm(Package):
     homepage = "http://dri.freedesktop.org/libdrm/"
     url      = "http://dri.freedesktop.org/libdrm/libdrm-2.4.59.tar.gz"
 
+    list_url = "https://dri.freedesktop.org/libdrm"
```

Comment:
This line isn't necessary. We already add the basename of `url` to the search path.